src/pages/Todo/TodoItem.js

```diff
@@ -13,8 +13,8 @@ const TodoItem = ({ item, setStale }) => {
         Server.collectionID,
         item["$id"],
         data,
-        item["$permissions"]["read"],
-        item["$permissions"]["write"]
+        item["read"],
```

Comment:
Should these be `$read` and `$write`?
```suggestion
        item["$read"],
```

Reply:
Oh yes you're right :') Thanks for the catch!